template.yaml

```diff
@@ -145,21 +145,23 @@ Resources:
             title: NVA Resource
             type: object
             properties:
+              type:
+                type: string
               identifier:
                 type: string
                 format: uuid
               createdDate:
                 type: string
-                format: date-time
+                pattern: '^([0-9]+)-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])[Tt]([01][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9]|60)(\.[0-9]+)?(([Zz])|([\+|\-]([01][0-9]|2[0-3]):[0-5][0-9]))$' #RFC3339
```

Comment:
maybe add a TODO comment to follow up this issue so that if it is resolved to put back date-time?